src/System.Net.Security/ref/System.Net.Security.csproj

```diff
@@ -2,9 +2,12 @@
   <PropertyGroup>
     <ProjectGuid>{A7488FC0-9A8F-4EF9-BC3E-C5EBA47E13F8}</ProjectGuid>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;uap-Debug;uap-Release</Configurations>
+    <!-- This is needed so that code for TlsCipherSuite will have no namespace (causes compile errors) when used within T4 template  -->
+    <DefineConstants>PRODUCT</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Net.Security.cs" />
+    <Compile Include="TlsCipherSuite.cs" />
```

Comment:
I don't think this needs to be here.  As part of this PR you should just run `msbuild /t:GenerateReferenceSource` from the src directory (after building the src directory); which will make the enum exist with all of its declared members.  (Editing it into System.Net.Security.cs).

Reply:
@bartonjs fine except the it generated APIs like:

```diff
+        TLS_AES_256_GCM_SHA384 = (ushort)4866,
+        TLS_CHACHA20_POLY1305_SHA256 = (ushort)4867,
+        TLS_DHE_DSS_EXPORT_WITH_DES40_CBC_SHA = (ushort)17,
+        TLS_DHE_DSS_WITH_3DES_EDE_CBC_SHA = (ushort)19,
+        TLS_DHE_DSS_WITH_AES_128_CBC_SHA = (ushort)50,
```

let me know if that is acceptable

Reply:
That's fine.  The important thing with the ref is that we see when things are being added (or, worse, attempting a delete).  Since the generator will generate them the same way every time it all works out in the end.

Reply:
@bartonjs see my 2 comments on the next commit - finalizer seems fine (but weird) but lack of default argument seems worrysome